### ISSUE TITLE
Feature request/3rd party tool need: allcontributors-to-zenodo

### ISSUE THREAD

**Issue by yarikoptic** [all-contributors/all-contributors#418]
Outlined the idea in https://github.com/con/allcontributors-to-zenodo/blob/master/README.md and sent out an original call in https://github.com/con/allcontributors-to-zenodo/issues/1 .  If it would be magically supported by allcontributors "centrally" -- that would be just awesome, but I did not want to request a feature which might not be of interest to the developers here.

**Comment by Berkmann18** [comment]
As @gr2m pointed out in #301, it's kind of out of scope and I see how it would be useful for research projects however, that wouldn't necessarily be the case for everyone.

That being said, after looking at @adswa's initial paragraph and https://github.com/all-contributors/all-contributors/issues/290, it seems that the custom template feature (which in theory would allow for this feature to be implemented) simply doesn't work properly according to:
> It is not currently working, you are correct.
> 
> PRs welcome to support this!
> -- <cite>@jakebolam</cite>

So I'm wondering if you tried to use the custom template options for either the [CLI](https://allcontributors.org/docs/en/cli/configuration) or the [bot](https://allcontributors.org/docs/en/bot/configuration)?
If not, then that should enable you to have the Zenodo fields included in the AC configuration without relying on a 3rd party or plugin.

For what it's worth, if the custom templating system (even when it works) isn't suitable for this, then a plugin/3rd-party tool/package would be welcome (IMHO).

**Comment by yarikoptic** [comment]
FWIW, @vsoch made a huge initial leap in establishing such 3rd party tool which would also assist in establishing the initial list of contributors based on git commits history (WiP in https://github.com/con/allcontributors-zenodo/pull/2/) (if only there was a Python library for allcontributors client operations ... ;)) and later assisting with auto-linking (I still hope it should be possible in the majority of cases) to zenodo.

**Comment by vsoch** [comment]
Yes! But the organization and interaction needs a bit of a refactor, it’s not quite up to my quality bar yet. But I should have some time to continue work this week.
